discovery/discovery.go

```diff
@@ -10,23 +14,97 @@ import (
 	"github.com/golang/glog"
 )
 
-type offchainOrchestrator struct {
-	uri   *url.URL
+const GetOrchestratorsTimeoutLoop = 3 * time.Second
+
+type offchainOrchestrators struct {
+	uri   []*url.URL
 	bcast server.Broadcaster
 }
 
-func NewOffchainOrchestrator(node *core.LivepeerNode, address string) *offchainOrchestrator {
-	addr := "https://" + address
-	uri, err := url.ParseRequestURI(addr)
+func NewOffchainOrchestrator(node *core.LivepeerNode, addresses []string) *offchainOrchestrators {
+	var uris []*url.URL
+
+	for _, addr := range addresses {
+		if !strings.HasPrefix(addr, "http") {
+			addr = "https://" + addr
+		}
+		uri, err := url.ParseRequestURI(addr)
+		if err != nil {
+			glog.Error("Could not parse orchestrator URI: ", err)
+			continue
+		}
+		uris = append(uris, uri)
+	}
+
+	bcast := core.NewBroadcaster(node)
+	return &offchainOrchestrators{bcast: bcast, uri: uris}
```

Comment:
I wonder if we should log something, somewhere, if `uris` is empty, just to notify the user.

Reply:
Sure, that sounds reasonable!